internal/node/node.bzl

```diff
@@ -162,14 +162,21 @@ def _nodejs_binary_impl(ctx):
         "TEMPLATED_repository_args": _short_path_to_manifest_path(ctx, ctx.file._repository_args.short_path),
         "TEMPLATED_script_path": script_path,
     }
+
+    files = [node, ctx.outputs.loader, ctx.file._repository_args] + ctx.files._node_runfiles
```

Comment:
files is not a very descriptive identifier here. most things in bazel are files.

Reply:
Humm, I admit, this var name is awful.